calculator/calculator_test.go

```diff
@@ -109,20 +109,9 @@ func TestDivideTable(t *testing.T) {
 	}
 }
 
-// func TestDividebyZero(t *testing.T){
-// data := []struct {
-// 	a int
-// 	b int
-// 	want int
-// } {
-// 	{4, 0, 0},
-// 	{6, 0, 0},
-// 	{8, 0, 0},
-// }
-// for _, val := range data{
-// 	got := calculator.TestDividebyZero(val.a, val.b)
-// 	if got != val.want {
-// 		t.Errorf("got %d want %d", got, val.want)
-// 	}
-// }
-// }
+func TestDivideByZero(t *testing.T) {
+	_, ok := calculator.DividebyZero(4, 0)
```

Comment:
1. You should capture the return result and check that what you got is what you wanted
2. Your error variable should be more appropriately named (i.e. `err`) and you should be checking whether it's `nil` and failing the test if what you got is not what you expected.
3. You should be using table-driven tests